### ISSUE TITLE
Reset Console encountered problem

### ISSUE THREAD

**Issue by MikeHsuTW** [spyder-ide/spyder#18554]
### What steps will reproduce the problem?

--- Logging error ---
Traceback (most recent call last):
  File "C:\Anaconda3\lib\asyncio\events.py", line 81, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Anaconda3\lib\site-packages\tornado\platform\asyncio.py", line 579, in _handle_select
    self._handle_event(r, self._readers)
  File "C:\Anaconda3\lib\site-packages\tornado\platform\asyncio.py", line 591, in _handle_event
    callback()
  File "C:\Anaconda3\lib\site-packages\tornado\platform\asyncio.py", line 189, in _handle_events
    handler_func(fileobj, events)
  File "C:\Anaconda3\lib\site-packages\zmq\eventloop\zmqstream.py", line 445, in _handle_events
    zmq_events = self.socket.EVENTS
  File "C:\Anaconda3\lib\site-packages\zmq\sugar\attrsettr.py", line 51, in __getattr__
    return self._get_attr_opt(upper_key, opt)
  File "C:\Anaconda3\lib\site-packages\zmq\sugar\attrsettr.py", line 63, in _get_attr_opt
    return self.get(opt)
  File "C:\Anaconda3\lib\site-packages\zmq\_future.py", line 215, in get
    result = super(_AsyncSocket, self).get(key)
  File "zmq\backend\cython\socket.pyx", line 464, in zmq.backend.cython.socket.Socket.get
  File "zmq\backend\cython\socket.pyx", line 135, in zmq.backend.cython.socket._check_closed
zmq.error.ZMQError: not a socket

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Anaconda3\lib\logging\__init__.py", line 1088, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "C:\Anaconda3\lib\threading.py", line 890, in _bootstrap
    self._bootstrap_inner()
  File "C:\Anaconda3\lib\threading.py", line 932, in _bootstrap_inner
    self.run()
  File "C:\Anaconda3\lib\site-packages\jupyter_client\threaded.py", line 218, in run
    self.ioloop.start()
  File "C:\Anaconda3\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "C:\Anaconda3\lib\site-packages\nest_asyncio.py", line 72, in run_forever
    self._run_once()
  File "C:\Anaconda3\lib\site-packages\nest_asyncio.py", line 119, in _run_once
    handle._run()
  File "C:\Anaconda3\lib\asyncio\events.py", line 95, in _run
    self._loop.call_exception_handler(context)
  File "C:\Anaconda3\lib\asyncio\base_events.py", line 1733, in call_exception_handler
    self.default_exception_handler(context)
  File "C:\Anaconda3\lib\asyncio\base_events.py", line 1707, in default_exception_handler
    logger.error('\n'.join(log_lines), exc_info=exc_info)
Message: 'Exception in callback AddThreadSelectorEventLoop._handle_select([6640], [])\nhandle: <Handle AddThreadSelectorEventLoop._handle_select([6640], [])>'
Arguments: ()

### Traceback
```python-traceback
--- Logging error ---
Traceback (most recent call last):
  File "C:\Anaconda3\lib\asyncio\events.py", line 81, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Anaconda3\lib\site-packages\tornado\platform\asyncio.py", line 579, in _handle_select
    self._handle_event(r, self._readers)
  File "C:\Anaconda3\lib\site-packages\tornado\platform\asyncio.py", line 591, in _handle_event
    callback()
  File "C:\Anaconda3\lib\site-packages\tornado\platform\asyncio.py", line 189, in _handle_events
    handler_func(fileobj, events)
  File "C:\Anaconda3\lib\site-packages\zmq\eventloop\zmqstream.py", line 445, in _handle_events
    zmq_events = self.socket.EVENTS
  File "C:\Anaconda3\lib\site-packages\zmq\sugar\attrsettr.py", line 51, in __getattr__
    return self._get_attr_opt(upper_key, opt)
  File "C:\Anaconda3\lib\site-packages\zmq\sugar\attrsettr.py", line 63, in _get_attr_opt
    return self.get(opt)
  File "C:\Anaconda3\lib\site-packages\zmq\_future.py", line 215, in get
    result = super(_AsyncSocket, self).get(key)
  File "zmq\backend\cython\socket.pyx", line 464, in zmq.backend.cython.socket.Socket.get
  File "zmq\backend\cython\socket.pyx", line 135, in zmq.backend.cython.socket._check_closed
zmq.error.ZMQError: not a socket

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Anaconda3\lib\logging\__init__.py", line 1088, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "C:\Anaconda3\lib\threading.py", line 890, in _bootstrap
    self._bootstrap_inner()
  File "C:\Anaconda3\lib\threading.py", line 932, in _bootstrap_inner
    self.run()
  File "C:\Anaconda3\lib\site-packages\jupyter_client\threaded.py", line 218, in run
    self.ioloop.start()
  File "C:\Anaconda3\lib\site-packages\tornado\platform\asyncio.py", line 199, in start
    self.asyncio_loop.run_forever()
  File "C:\Anaconda3\lib\site-packages\nest_asyncio.py", line 72, in run_forever
    self._run_once()
  File "C:\Anaconda3\lib\site-packages\nest_asyncio.py", line 119, in _run_once
    handle._run()
  File "C:\Anaconda3\lib\asyncio\events.py", line 95, in _run
    self._loop.call_exception_handler(context)
  File "C:\Anaconda3\lib\asyncio\base_events.py", line 1733, in call_exception_handler
    self.default_exception_handler(context)
  File "C:\Anaconda3\lib\asyncio\base_events.py", line 1707, in default_exception_handler
    logger.error('\n'.join(log_lines), exc_info=exc_info)
Message: 'Exception in callback AddThreadSelectorEventLoop._handle_select([6640], [])\nhandle: <Handle AddThreadSelectorEventLoop._handle_select([6640], [])>'
Arguments: ()
```

## Versions

* Spyder version: 5.0.3 
* Python version: 3.8.13
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.3 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0               :  8.3.0 (OK)
jedi =0.17.2                  :  0.17.2 (OK)
jsonschema >=3.2.0            :  4.4.0 (OK)
keyring >=17.0.0              :  23.4.0 (OK)
nbconvert >=4.0               :  6.4.4 (OK)
numpydoc >=0.6.0              :  1.2 (OK)
paramiko >=2.4.0              :  2.8.1 (OK)
parso =0.7.0                  :  0.7.0 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.9.0 (OK)
pygments >=2.0                :  2.11.2 (OK)
pylint >=1.0                  :  2.12.2 (OK)
pyls >=0.36.2;<1.0.0          :  0.36.2 (OK)
pyls_black >=0.4.6            :  0.4.6 (OK)
pyls_spyder >=0.3.2;<0.4.0    :  0.3.2 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.1.10 (OK)
qtawesome >=1.0.2             :  1.0.3 (OK)
qtconsole >=5.1.0             :  5.3.0 (OK)
qtpy >=1.5.0                  :  2.0.1 (OK)
rtree >=0.9.7                 :  0.9.7 (OK)
setuptools >=39.0.0           :  61.2.0 (OK)
sphinx >=0.6.6                :  4.4.0 (OK)
spyder_kernels >=2.0.3;<2.1.0 :  2.0.3 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3;<2.0.0      :  1.0.2 (OK)
zmq >=17                      :  22.3.0 (OK)

# Optional:
cython >=0.21                 :  0.29.28 (OK)
matplotlib >=2.0.0            :  3.5.1 (OK)
numpy >=1.7                   :  1.21.5 (OK)
pandas >=1.1.1                :  1.4.2 (OK)
scipy >=0.17.0                :  1.7.3 (OK)
sympy >=0.7.3                 :  1.10.1 (OK)
```

**Comment by ccordoba12** [comment]
Hey @MikeHsuTW, thanks for reporting. I think this error is fixed in our latest version (5.3.1).

Since it's still not available in Anaconda, you have two options:

1. Use our Windows installer, which you can find [here](https://github.com/spyder-ide/spyder/releases).
2. Create a new environment with conda-forge packages. For that, please close Spyder, open the Anaconda Prompt and run the following commands there:

        conda create -n spyder-cf -c conda-forge spyder
        conda activate spyder-cf
        spyder

Please do that and let us know if it works for you.

**Comment by ccordoba12** [comment]
Closing due to lack of response.
